### ISSUE TITLE
Clarification on How To Use Learning rate auto scaling

### ISSUE THREAD

**Issue by hnyll** [open-mmlab/mmdetection#11497]
(lets say i have 3 gpus)
(for grounding_dino training. custom coco dataset)

if i enable learning rate auto scaling by adding the '--auto-scale-lr' argument when training then

A)  Learning Rate

1. Do I need to change my base learning rate? (if i read it correctly, i dont need to change this if i enable auto scaling feature)
2. is the base learning rate 0.0005? if not, what is it?

B) Batch SIze

1. Do i need to change the batch size? (if i read it correctly, i dont need to change this if i enable auto scaling feature)
2. if so where can i find samples_per_gpu? or do i need to specify in my dataloader?
3. what is num_workers vs samples_per gpu? (i thought num_workers = samples_per_gpu but someone explain if im wrong)

just confused on if i need to change my variables (base_batch_size, lr) or leave them for the code to figure out when auto-scale-lr option is enabled. and if i do need to change my variables, i want to know where i need to go to change them

Thanks in advance
let me know if you need any further details about my issue
